docs/instrumentation-list.yaml

```diff
@@ -302,18 +396,26 @@ executors:
   instrumentations:
   - name: executors
     srcPath: instrumentation/executors
+    scope:
+      name: io.opentelemetry.executors
     target_versions:
-      javaagent: []
+      javaagent:
+      - Java 8+
 external:
   instrumentations:
   - name: external-annotations
     srcPath: instrumentation/external-annotations
+    scope:
+      name: io.opentelemetry.external-annotations
     target_versions:
-      javaagent: []
+      javaagent:
+      - Java 8+
```

Comment:
this one is targeting specific baseline versions of the different annotation libraries

Reply:
i wonder if we should include these types of instrumentations in this context, maybe a separate analyzer and output for things like this and the `methods` instrumentation, they seem like a different "class" of instrumentation

Reply:
I have same (lack of) thought as https://github.com/open-telemetry/opentelemetry-java-instrumentation/pull/13531/files#r2003831509

Reply:
my current thinking is that I'll mark them in a way that I can exclude them for now, and circle back with some different way of approaching things that do not instrument a library or the JDK